.github/workflows/ci-cd.yml

```
@@ -175,22 +175,38 @@ jobs:
   # ------------------------------------------------------------------
   # 🎨 VISUAL REGRESSION (Chromatic)
   # Catches UI changes in Storybook components.
+<<<<<<< HEAD
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)